FiducialRegistrationWizard/qSlicerFiducialRegistrationWizardModuleWidget.cxx

```diff
@@ -148,7 +148,7 @@ std::string qSlicerFiducialRegistrationWizardModuleWidget::GetCorrespondingFiduc
   }
 
   std::stringstream correspondingFiducialString;
-  correspondingFiducialString << "Place fiducial (";
+  correspondingFiducialString << qSlicerFiducialRegistrationWizardModuleWidget::tr("Place fiducial ");
```

Comment:
This needs to be changed so that a complete sentence is translated, not just fragments. Something like `tr("Place fiducial (corresponding to %1")`